executor/hnswpsql.py

```diff
@@ -239,10 +239,22 @@ def status(self, **kwargs):
         :return: DocumentArray with one Document with tags 'psql_docs', 'hnsw_docs',
         'last_sync', 'pea_id'
         """
-        psql_docs = self._kv_indexer.size
-        hnsw_docs = self._vec_indexer.size
-        last_sync = self._vec_indexer.last_timestamp
-        last_sync = last_sync.isoformat()
+        psql_docs = None
+        hnsw_docs = None
+        last_sync = None
+
+        if self._kv_indexer and self._kv_indexer.initialized:
+            psql_docs = self._kv_indexer.size
+        else:
+            self.logger.warning(f'PSQL connection has not been initialized')
+
+        if self._vec_indexer:
+            hnsw_docs = self._vec_indexer.size
+            last_sync = self._vec_indexer.last_timestamp
+            last_sync = last_sync.isoformat()
```

Comment:
```suggestion
            last_sync = self._vec_indexer.last_timestamp.isoformat()
```

Reply:
too minor a change to justify the rebasing pain into the PR that depends on this.